taxi/urls.py

```diff
@@ -13,12 +13,18 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverUpdateView,
+    DriverDeleteView, assign_driver_to_car, delete_driver_from_car,
```

Comment:
bad code style, make all views from the new line

Reply:
Fixed